src/utils.ts

```diff
@@ -77,7 +77,15 @@ export function openFileInEditor(fileName: string, msg: string): string {
 
         writeFileSync(filePath, msg)
 
-        const editor = spawnSync('git', ['config', '--global', 'core.editor']).stdout
+        const editor = process.env.EDITOR
+            ? process.env.EDITOR
+            : process.env.VISUAL
+            ? process.env.VISUAL
+            : spawnSync('git', ['config', '--global', 'core.editor']).stdout
+
```

Comment:
Is your logic trying to say, use `EDITOR`, if that's not there fallback to `VISUAL`, if that's not there get it from git?

If so can we clean it up like:

```javascript
const editor = process.env.EDITOR || process.env.VISUAL || spawnSync('git', ['config', '--global', 'core.editor']).stdout
```

Reply:
@protoEvangelion sorry for the late reply, time zone differences are hard to deal with.

I'm going to change this as you suggested, as for the failing test, I'm guessing it's because I'm throwing an error.
I'll push my changes and see what happens - locally no tests fail.

Reply:
Thanks for the quick turnaround on this! I'm updating the docs & then I will merge this.

Reply:
Thanks to you, no worries on my side it was fun contributing to a project I use daily at work

Reply:
Awesome, let me know if you have more ideas. I'm happy to chat 👍